lib/files.js

```diff
@@ -4,6 +4,11 @@ const { getEnv } = require('@hubspot/local-dev-lib/config');
 const { getHubSpotWebsiteOrigin } = require('./urls');
 const { ENVIRONMENTS } = require('./constants');
 
+/**
+ * @deprecated
+ * Use the corresponding export from local-dev-lib
+ * https://github.com/HubSpot/hubspot-local-dev-lib
+ */
```

Comment:
renamed to `@hubspot/local-dev-lib/cms/themes`

Reply:
added in [a821417](https://github.com/HubSpot/cli-lib/pull/39/commits/a821417f90003cb28b571909a63f011d96365049)